Runtime/Editor/ProjectSettings.cs

```diff
@@ -91,6 +95,8 @@ private void DrawGameSettings()
             if (EditorGUI.EndChangeCheck())
             {
                 gameSettings.domainKey = m_CustomSettings.FindProperty("domainKey").stringValue;
+                DomainKeyEnteredEvent?.Invoke();
+                m_CustomSettings.FindProperty("domainKey").stringValue = gameSettings.domainKey;
```

Comment:
From what I understand m_CustomSettings will be recreated from gameSettings next frame right? So why set this?